cmds/vboot/vboot.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"crypto/sha1"
+	"crypto/sha256"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/zaolin/go-tpm/tpm"
+	"golang.org/x/crypto/ed25519"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	tpmDevice  string = "/dev/tpm0"
+	mountPath  string = "/mnt/vboot"
+	filesystem string = "ext3"
+)
+
+func dieHard() {
+	if e1, e2, err := syscall.Syscall6(syscall.SYS_REBOOT, syscall.LINUX_REBOOT_MAGIC1, syscall.LINUX_REBOOT_MAGIC2, syscall.LINUX_REBOOT_CMD_POWER_OFF, 0, 0, 0); err != 0 {
+		log.Fatalf("a %v b %v err %v", e1, e2, err)
+	}
+}
+
+func main() {
+	var publicKey = flag.String("pubkey", "/etc/sig.pub", "A public key which should verify the signature.")
```

Comment:
Please declare flags outside of main at the package level, and group them, below the consts:

```go
var (
   pcr = flag.Uint(...
   bootDev = flag.....
)
```